src/modules/infobox/components/FeatureDetail.vue

```diff
@@ -0,0 +1,48 @@
+<script setup>
+import DOMPurify from 'dompurify'
+import { computed, toRefs } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+import SelectableFeature from '@/api/features/SelectableFeature.class.js'
+
+const props = defineProps({
+    feature: {
+        type: SelectableFeature,
+        required: true,
+    },
+})
+
+const { feature } = toRefs(props)
+
+const i18n = useI18n()
+
+const hasFeatureStringData = computed(() => typeof feature.value?.data === 'string')
+
+const featureDataEntries = computed(() => {
+    if (hasFeatureStringData.value) {
+        return []
+    }
+    return Object.entries(feature.value.data).filter(([_, value]) => value)
+})
+function sanitizeHtml(htmlText) {
+    return DOMPurify.sanitize(htmlText)
+}
```

Comment:
Would it not make more sense to use a computed here instead of a method ? This would improve performance.

Reply:
I changed a bit how it's done, I sanitize the content of JSON data in the computed, and when the data is a String but doesn't come from our (trusted) backends.
I don't think I can make a computed out of every use case, so I kept the function as it is here